packages/project-utils/bundling/app/config/webpack.config.js

```diff
@@ -40,9 +40,11 @@ const cssRegex = /\.css$/;
 const cssModuleRegex = /\.module\.css$/;
 const sassRegex = /\.(scss|sass)$/;
 const sassModuleRegex = /\.module\.(scss|sass)$/;
+
 const sassLoader = {
     loader: require.resolve("sass-loader"),
     options: {
+        // webpackImporter: false,
```

Comment:
Leave a comment why this is commented out.

Reply:
It should be removed, I was doing a test to see if it materially changed anything but it didn't